cves/CVE-2002-0661.yml

```diff
@@ -199,4 +224,13 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    The code mistake made here was only checking for '/' when sanitizing a 
+    potential path traversal vulnerability. If I try to put myself into the 
+    minds of the developers, I imagine they, likely with mostly UNIX experience, 
+    simply did not consider that backslashes could be used for path traversal 
+    in other operating systems. Even one developer who looked at and commented 
+    on the code in question between the VCC and the fix did not catch the issue 
+    before release. This shows the importance of thoroughly testing and viewing 
+    your software from different perspectives--perspectives which might reveal 
+    new problems.
```

Comment:
Consider slapping a newline onto the end of this file: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline